src/tests/ftest/avocado-teardown-timeout.patch

```diff
@@ -47,7 +47,7 @@ index 104a7dac..ca349e98 100644
                          os.kill(proc.pid, signal.SIGTERM)
                      except OSError:
                          pass
--                    break
+                     break
                  wait.wait_for(lambda: not queue.empty() or not proc.is_alive(),
                                cycle_timeout, first, step)
                  if test_status.interrupt:
```

Comment:
Does this patch actually apply without any errors, maybe about trailing cruft, etc?  I would be surprised if it does.  I think you actually want:
```suggestion
```
here as these lines are outside of the `7`/`12` in the new patch.

Reply:
I just used the output of the diff after manually making the change.  The patch was successful:
```
[2021-02-11T01:19:26.594Z] + sudo patch -p1 -d /usr/lib/python2.7/site-packages/
[2021-02-11T01:19:26.594Z] patching file avocado/core/runner.py
[2021-02-11T01:19:26.594Z] + grep TIMEOUT_TEARDOWN /usr/lib/python2.7/site-packages//avocado/core/runner.py
[2021-02-11T01:19:26.594Z] + cat
[2021-02-11T01:19:26.594Z] + sudo patch -p1 -d /usr/lib/python2.7/site-packages/
[2021-02-11T01:19:26.594Z] patching file avocado/core/runner.py
[2021-02-11T01:19:26.594Z] patching file avocado/core/test.py
[2021-02-11T01:19:26.594Z] + grep 'def phase(self)' /usr/lib/python2.7/site-packages//avocado/core/test.py
[2021-02-11T01:19:26.594Z] + filterdiff -p1 -x 'selftests/*'
[2021-02-11T01:19:26.594Z] + sed -e '/selftests\/.*/d'
[2021-02-11T01:19:26.594Z] + sudo patch -p1 -d /usr/lib/python2.7/site-packages/
[2021-02-11T01:19:26.594Z] patching file avocado/core/test.py
```
And it worked:
```
[2021-02-11T01:20:29.523Z] JOB LOG    : /var/tmp/ftest/avocado/job-results/job-2021-02-11T01.20-0a2fc9e/job.log
[2021-02-11T01:20:35.053Z]  (1/4) ./harness/test_signal_handler.py:TestSignalHandler.test_sigterm;hosts-sigterm_no_delay-complete_test-f6c7:  PASS (5.01 s)
[2021-02-11T01:20:40.372Z]  (2/4) ./harness/test_signal_handler.py:TestSignalHandler.test_sigterm;hosts-sigterm_delay-complete_test-12f9:  PASS (5.01 s)
[2021-02-11T01:20:50.350Z]  (3/4) ./harness/test_signal_handler.py:TestSignalHandler.test_sigterm;hosts-sigterm_no_delay-timeout_test-8928:  INTERRUPTED (10.28 s)
[2021-02-11T01:21:02.814Z]  (4/4) ./harness/test_signal_handler.py:TestSignalHandler.test_sigterm;hosts-sigterm_delay-timeout_test-1fe8:  INTERRUPTED (10.27 s)
[2021-02-11T01:21:02.814Z] RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 2 | CANCEL 0
```